tests/ci/build-pkg.yml

```diff
@@ -0,0 +1,91 @@
+steps:
+- id: 'create .git/ folder'
+  name: 'alpine/git:v2.24.1'
+  entrypoint: '/bin/sh'
+  args:
+  - '-ec'
+  - |
+    if [[ ! -d ".git/" ]]; then
+      git init
+      git remote add origin https://github.com/vectorizedio/redpanda
+
+      echo "Checking out COMMIT=${COMMIT_SHA}"
+      git fetch --depth=200 origin $COMMIT_SHA
+      git reset --hard FETCH_HEAD
+
+      # fetch tags. if no commit found, fetch more.
+      # fetch everything if tag is still not found.
+      git fetch --tags origin
+      if ! git describe; then
+        git fetch --deepen=500 origin
+        if ! git describe; then
+          git fetch --unshallow origin || true
+        fi
+      fi
+    fi
+
+    if [[ "$_GITHUB_API_TOKEN" == "na" ]]; then
+      echo "Expecting value for _GITHUB_API_TOKEN"
+      exit 1
+    fi
+    git clone --depth 1 -b dev https://$_GITHUB_API_TOKEN@github.com/vectorizedio/vtools vtools
+
+- id: 'install vtools'
+  name: 'gcr.io/redpandaci/fedora:31-python37-docker1903'
+  entrypoint: bash
+  args:
+  - -ec
+  - |
+    python3 -mvenv --system-site-packages build/venv/v
```

Comment:
do we need `venv` anymore ?

Reply:
Installing in a venv (so in a specific directory) is an easy way to share the vtools installation between different build steps (which are different containers). If we installed in the system, we would have to reinstall vtools in each step.